bedrock/newsletter/templates/newsletter/existing.html

```diff
@@ -1,3 +1,7 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
Not a blocker, but the page margins for the preferences page could perhaps do with a little tidying up? The content doesn't seem to be centered in the page, and there is very little padding at the bottom of the page below the submit button:

![image](https://user-images.githubusercontent.com/400117/71969781-4870e080-31ff-11ea-8540-d8afb075b7c6.png)